docs/api-testing/project-access.md

```diff
@@ -1,66 +1,85 @@
+- If access is assigned to a team and you need to allow organization access to the project, you can manage this change using the previous process for changing from organization to team. You need to be a team member of the team the project access is assigned to and a team member of the team for the new project or an organization admin to do this.
 
 ## Email Notifications
 
-:::note IMPORTANT
-The email notification mechanism we have in place consist in an email notification when the test starts failing (the subject of the email will contain "Incident started"). A new email notification will be sent after 30 minutes if the test is still failing (the subject of the email will contain "Incident in progress"). This notification is sent only if your tests run less then every 30 minutes. As soon as the test execution passes, you will be notified via email (the subject of the email will contain "Incident resolved").
-:::
+You can configure one or more email addresses to be notified when tests in a project are failing. 
+
+Email notifications are triggered for failing tests. When a test starts failing, an email with "Incident started" included in the subject line is sent to each address configured to receive notifications for a project. A new email notification will be sent after 30 minutes if the test is still failing. The follow up email will include "Incident in progress" in the subject line. This notification is sent only if tests run less than every 30 minutes. When test execution passes, an email is sent with "Incident resolved" in the subject line. 
 
 To enable email notifications:
 
 1. In **API Testing**, on the **Projects** page, under a project name, click **Settings**.
 2. In the left nav, click **Notifications**.
-3. In the email address field, enter an address and then click **Save changes**.<br/><img src={useBaseUrl('img/api-fortress/2021/02/dash-1.5.png')} alt="Dashboard Pic 1.5"/>
+3. In the email address field, enter an address. To add more email addresses, click **+ New Email**. 
+5. Click **Save changes**. 
+<br/><img src={useBaseUrl('img/api-testing/notifications.png')} alt="Dashboard Pic 1.5"/>
 
 :::tip
 Use the Fact component in your API tests to control the behavior of email notifications. This can be useful if you're testing in multiple environments. For more information, see [Fact](/api-testing/composer/other-components/#fact).
 :::
 
 ## Connectors
 
-See [Connectors](/api-testing/integrations/pagerduty-webhooks/).
+See [PagerDuty and Webhook Connectors](/api-testing/integrations/pagerduty-webhooks/) for more information.
 
 ## Deleting a Project
```

Comment:
We should be consistent with the use of gerunds vs. non-gerunds for task headings. 

For example: Deleting a Project vs. Delete a Project -- the heading above uses the non-gerund (Create a Project). I checked our style guide and it doesn't contain any explicit guidelines in this regard -- perhaps this is a candidate style topic for discussion.